Oryx.sln

```diff
@@ -26,15 +24,19 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "build", "build", "{0FEBE7DD-10C0-4650-89E4-69DDB0930C56}"
 	ProjectSection(SolutionItems) = preProject
 		build\__variables.sh = build\__variables.sh
-		build\build-buildimages.sh = build\build-buildimages.sh
-		build\build-constants.yaml = build\build-constants.yaml
-		build\build-runtimeimages.sh = build\build-runtimeimages.sh
-		build\build-solution.sh = build\build-solution.sh
-		build\build-testimage.sh = build\build-testimage.sh
-		build\test-buildimages.sh = build\test-buildimages.sh
-		build\test-buildscriptgenerator.sh = build\test-buildscriptgenerator.sh
-		build\test-integration.sh = build\test-integration.sh
-		build\test-runtimeimages.sh = build\test-runtimeimages.sh
+		build\buildBuildImageBases.sh = build\buildBuildImageBases.sh
+		build\buildBuildImages.sh = build\buildBuildImages.sh
+		build\buildBuildpacksImages.sh = build\buildBuildpacksImages.sh
+		build\buildRunTimeImageBases.sh = build\buildRunTimeImageBases.sh
+		build\buildRunTimeImages.sh = build\buildRunTimeImages.sh
+		build\buildSln.sh = build\buildSln.sh
+		build\buildTestBuildImage.sh = build\buildTestBuildImage.sh
+		build\constants.yaml = build\constants.yaml
+		build\testBuildImages.sh = build\testBuildImages.sh
+		build\testBuildScriptGenerator.sh = build\testBuildScriptGenerator.sh
+		build\testIntegration.sh = build\testIntegration.sh
+		build\testRunTimeImages.sh = build\testRunTimeImages.sh
+		build\testStartupScriptGenerators.sh = build\testStartupScriptGenerators.sh
```

Comment:
these changes are probably not that important, but being a vs user when you open oryx.sln and you try to open some files (bash or yaml) in your repo and it keeps giving you annyoying error that file has been moved or deleted. I only included what we had earlier. We have many more, we can add them  if we want. But I guess this is as minimum as it can be.